### DISCUSSION TITLE
CTKscrollableframe, how can i make the scrollbar scroll to the end automatically?

### DISCUSSION THREAD

**Discussion by EluarIsufaj** [TomSchimansky/CustomTkinter#1970] [Q&A] [answered]
Hi, I am going to be as straightforward as possible, so I'm making a chat app and I the messages are represented as labels in a scrollable frame, so I need the scrollbar to scroll to the end automatically when I .grid a new label, so if there is any way to do that please just share the solution! Thank you

**Comment by Akascape** [comment, marked as answer]
@EluarIsufaj Try this:
```python
scrollable_frame._parent_canvas.yview_moveto(1.0)
```

**Reply by EluarIsufaj** [reply]
@Akascape  Thank you!!!
